tests/devices/i04/test_beam_centre.py

```diff
@@ -0,0 +1,101 @@
+from collections.abc import AsyncGenerator
+from unittest.mock import MagicMock, patch
+
+import cv2
+import numpy as np
+import pytest
+from ophyd_async.core import init_devices
+
+from dodal.devices.i04.beam_centre import CentreEllipseMethod, binary_img
+
```

Comment:
Good tests, I would probably add one that is the whole thing from realistic data through to an actual centre